src/Analysis/GWASV2/Components/SelectHare/SelectHareDropDown.test.jsx

```diff
@@ -0,0 +1,160 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import { Select } from 'antd';
+import { QueryClient, QueryClientProvider } from 'react-query';
+import SelectHareDropDown from './SelectHareDropDown';
+import ValidInitialState from '../../TestData/InitialStates/ValidInitialState';
+import ACTIONS from '../../Utils/StateManagement/Actions';
+import { SourceContextProvider } from '../../Utils/Source';
+import {
+  fetchConceptStatsByHareSubset,
+  useSourceFetch,
+} from '../../Utils/cohortMiddlewareApi';
+
+// Mock relevant API calls:
+jest.mock('../../Utils/cohortMiddlewareApi');
```

Comment:
This is great; will allow us to test lots of other functionality using this pattern!